fuel/datasets/hdf5.py

```diff
@@ -106,16 +106,20 @@ class H5PYDataset(Dataset):
          array
       4. ``stop`` : stop index (exclusive) of the split in the source
          array
-      5. ``available`` : boolean, ``False`` is this split is not available
+      6. ``indices`` : h5py.Reference, reference to a dataset containing
```

Comment:
Also here, I think, it should be  said that indices can be "null" which means that the `start` and `stop` attributes specify the subset